src/App.js

```diff
@@ -1242,6 +1241,42 @@ function runGameTick(initialGameState) {
     }
   });
 
+  /* 
+  Sandstorm
+
+  Start at 30 seconds, at 1 dmg
+  Tickrate is every 0.2 seconds
+  Increase by 1 every tick
+  */
+  // TODO: End game when it reaches 600 dmg per tick
+  const sandstorm_initial_tick = 30000;
+  const sandstorm_tickrate = 200;
+
+  let tickDamage = 0;
+  if (nextGameState.tick - sandstorm_initial_tick > 0) {
+    if (
+      (nextGameState.tick - sandstorm_initial_tick) % sandstorm_tickrate ===
+      0
+    ) {
+      tickDamage = Math.floor(
+        (nextGameState.tick - sandstorm_initial_tick) / sandstorm_tickrate
+      );
+    }
+  }
+  nextGameState.sandstorm_damage = tickDamage;
```

Comment:
Could you create a global object sandstormTicks where keys are the tick and value is the damage. Then this logic would just be to read this object with the nextGameState.tick and apply the damage.

Reply:
As a good practice, the less state you have, the better. In this case we can precompute all the damages and their ticks so we should do that. Less likely to have bugs in the long run.